themes/openy_themes/openy_rose/templates/paragraph/paragraph--small-banner.html.twig

```diff
@@ -46,6 +46,7 @@
   ]
 %}
 {% set color = '#' ~ content.field_prgf_color['#items'].entity.field_color.value %}
+{% set description_text = content.field_prgf_description.0['#text'] %}
```

Comment:
That's better, however `content.field_prgf_description.0['#text']` may be empty from the begining.

We should add condition here, when we setup description text value. `description_text` should be just like a flag, `true` or `false`.